src/modules/CRUD/components/Details/Read/AttachmentView/ImportFile/ImportFile.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Intent } from '@blueprintjs/core';
+import Uppy from '@uppy/core';
+import { Dashboard } from '@uppy/react';
+import { buildHeaders } from '@terralego/core/modules/Api';
+import locale from '@uppy/locales/src/fr_FR';
```

Comment:
I need some advice for this point because I have an i18n issue.
We can get the language code with react-i18next but I don't really know how to dynamically import the right language.
An idea could be to wrap `this.uppy` in the constructor with a dynamic import like the following example:

```jsx
import(`@uppy/locales/src/${lng}`).then(locale => {
  this.uppy = Uppy({
    locale,
    // etc..
  });
})
```

But if the value of `lng` doest not exist in `@uppy/locales/src/` the app crashes immediately.
It's too dangerous.

Need some help.

Reply:
How many locales should we support ?

If we just need fr/en, as Uppy locales are currently not so big (~5 Ko), web could maybe import both in a single object then provide only the good one to component (falling back to _en_ if unavailable).

```js
import localeFr from '@uppy/locales/src/fr_FR';
import localeEn from '@uppy/locales/src/en_US';

const locales = {
  fr: localeFr,
  en: localeEn,
  default: localeEn,
};
```

then in component:

```jsx
    const locale = locales[locales[lng] ? lng : 'default'];
```


You may also simply use the exemple you suggest but with adding a simple check:
```jsx
const UPPY_LOCALES = ['fr_FR', 'en_US', /* … */];
const validLng = UPPY_LOCALES.includes(lng) ? lng : 'en_EN';

import(`@uppy/locales/src/${validLng}`).then(locale => {
  this.uppy = Uppy({
    locale,
    // etc..
  });
})
```

Reply:
This seems a good approach, I chose the second.